### DISCUSSION TITLE
taking more than 3 hours for 50*50 distance matrix to find route with OR tools

### DISCUSSION THREAD

**Discussion by SruthyJohn** [google/or-tools#3090] [Routing (and legacy CP) questions]
Hi Team,
With  locations/stores my code was working fine and was giving routes, but when I tried to increase the matrix size it is running for 3 hours without any results.
Have included timewindows constraint and capacity constraint.

**Comment by lperron** [comment]
1) do you have a time limit ?
2) are you enabled local search ?
